FV3/gfsphysics/GFS_layer/GFS_driver.F90

```diff
@@ -894,6 +899,10 @@ subroutine GFS_phys_time_vary (Model, Grid, Tbd, Statein)
 
     nblks = size(blksz,1)
 
+    if ((Model%imp_physics == 99) .and. (Model%kdt == 1) .and. (.not. Model%restart))  then
```

Comment:
I realize you are copying logic from the old implementation, but it could make things slightly more readable to use this:
```suggestion
    if ((Model%imp_physics == Model%imp_physics_zhao_carr) .and. (Model%kdt == 1) .and. (.not. Model%restart))  then
```